### PR TITLE
tests: check for FAT32 ESP

### PR DESCRIPTION
**Pull request by bgilbert**
For https://github.com/coreos/coreos-assembler/pull/2491.

### CODE REVIEW

**Comment by dustymabe** [comment]
The current goal is to move away from `misc-ro` towards smaller and more scoped tests (see https://github.com/coreos/fedora-coreos-config/issues/1228). The primary benefit of `misc-ro` (running tests in a single VM) is now handled additionally by marking a test as `exclusive: false`. Would you mind breaking it out to a separately defined test with `exclusive: false`? Otherwise we'll do it as part of #1228.

**Comment by bgilbert** [comment]
Done!

File: tests/kola/storage/esp-fat32 [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# kola: { "exclusive": false }
```
Comment (jlebon) [inline tests/kola/storage/esp-fat32]: Shouldn't this test only run on arches which ship an ESP?

Comment (bgilbert) [reply tests/kola/storage/esp-fat32]: Blah.  Yes.

**Comment by bgilbert** [comment]
Updated.

**Review by jlebon — DISMISSED** [review]

File: tests/kola/storage/esp-fat32 [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# kola: { "exclusive": false, "architectures": "x86_64 aarch64" }
+# Check that the ESP is a FAT32 filesystem.
```
Comment (dustymabe) [inline tests/kola/storage/esp-fat32:3]:
```suggestion
# Check that the ESP is a FAT32 filesystem.
# https://github.com/coreos/fedora-coreos-tracker/issues/993
```

**Review by dustymabe — DISMISSED** [review]
LGTM - one suggestion

**Comment by dustymabe** [comment]
👍

**Review by dustymabe — APPROVED** [review]

**Comment by dustymabe** [comment]
and it's working! CI tests failed with:

```
fatal 'EFI System Partition is not FAT32'
```

because https://github.com/coreos/coreos-assembler/pull/2491 hasn't landed yet.

**Comment by bgilbert** [comment]
Closing per consensus in https://github.com/coreos/fedora-coreos-tracker/issues/993#issuecomment-943281115.

**Pull request closed without merging**